README.md

```diff
@@ -198,12 +198,11 @@ Select **Done** and then save your app.
     * If you're using a self-hosted private cloud or on-premises Zoom server, enter the **Zoom URL** and **Zoom API URL** for the Zoom server, for example `https://yourzoom.com` and `https://api.yourzoom.com/v2` respectively. Leave blank if you're using Zoom's vendor-hosted SaaS service.
   * **Cloud Hosted?**
     * Leave **Zoom API URL** and **Zoom URL** fields blank.
+* If you are using an account level app on Zoom, set **OAuth by Account Level App (Beta)** to `true`.
```

Comment:
We should probably get rid of references to "Beta" on this feature at this point, including in the plugin setting definition